Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -663,6 +692,10 @@ private void LateUpdate()
             if (IsActiveInstance)
             {
                 LateUpdateAllServices();
+                if (newProfile != null)
```

Comment:
Can you add some comments for why we're doing RemoveCurrentProfile here and then InitializeNewProfile in Update?

It's obvious to folks working on this right now but the order actually is worth documenting.